packages/client/src/components/ViewLogEntries/ViewLogEntries.tsx

```diff
@@ -20,22 +30,40 @@ export function ViewLogEntries() {
     logId: lastVisitedLog.id,
   });
   const [isCreateEntryOpen, setIsCreateEntryOpen] = useState(false);
+  const [isEditEntryOpen, setIsEditEntryOpen] = useState(false);
+  const [initialEditValues, setInitialEditValues] =
+    useState<EditLogEntryRequest>({
+      id: '' as unknown as Uuid,
```

Comment:
Whoever is picking this up next, if you have time could you figure out and clean up this as unknown portion? It felt wrong to add it but I was short on time.